ui/src/templates/api/index.ts

```diff
@@ -440,3 +440,15 @@ export const updateStackName = async (stackID, name) => {
 
   return resp
 }
+
+export const fetchReadMe = async (directory: string) => {
+  const resp = await fetch(
+    `https://raw.githubusercontent.com/influxdata/community-templates/master/${directory}/README.md`
+  )
+
+  if (resp.status >= 300) {
+    throw new Error('Network response was not ok' + resp.statusText)
```

Comment:
```suggestion
    throw new Error(`Network response was not ok: ${resp.statusText}`)
```

Reply:
ah thats my mistake actually, thanks for the fix!